app/src/main/kotlin/com/wire/android/ui/conversation/all/Conversation.kt

```diff
@@ -87,46 +82,20 @@ private fun ConversationListFloatingActionButton() {
         onClick = { })
 }
 
-@Composable
-private fun ConversationFolderHeader(name: String) {
-    Text(
-        text = name,
-        modifier = Modifier.padding(start = 16.dp, top = 8.dp, bottom = 8.dp),
-        maxLines = 1,
-        overflow = TextOverflow.Ellipsis,
-        style = MaterialTheme.typography.overline
-    )
-}
-
 @Composable
 private fun NewActivityRowItem(newActivity: NewActivity) {
     WhiteBackgroundWrapper(
         content = {
-            Row(
-                modifier = Modifier.padding(16.dp),
-                verticalAlignment = Alignment.CenterVertically,
-            ) {
-                with(newActivity.conversation.conversationInfo) {
-                    ConversationName(name)
-
-                    if (memberShip != Membership.None) {
-                        Spacer(modifier = Modifier.width(6.dp))
-                        MembershipQualifier(label = stringResource(id = memberShip.stringResource))
-                    }
-
-                    if (isLegalHold) {
-                        Spacer(modifier = Modifier.width(6.dp))
-                        LegalHoldIndicator()
-                    }
-                }
-
+            Row(verticalAlignment = Alignment.CenterVertically) {
+                UserInfoLabel(conversationInfo = newActivity.conversation.conversationInfo, Modifier.padding(16.dp))
                 Box(modifier = Modifier.fillMaxWidth()) {
                     EventBadgeFactory(
                         eventType = newActivity.eventType,
-                        modifier = Modifier.align(Alignment.CenterEnd)
+                        modifier = Modifier
+                            .align(Alignment.CenterEnd)
+                            .padding(end = 8.dp)
```

Comment:
why not getting the padding value from dimension resource instead of hardcoded ?

Reply:
how would you suggest to extract it and name it ?

Reply:
@MohamadJaara

Reply:
E.g. dimens.xml: 
```xml
<resources>
    <dimen name="small_padding">8dp</dimen>
</resources>
```
in code
```kotlin
Modifier.padding(dimensionResource(R.dimen.small_padding))
```

Reply:
good idea :) going to do that for now

Reply:
or maybe it is good idea to discuss it how we are going to approach paddings ?

Reply:
agreed on PM to change it later once we discuss it internal